src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -90,7 +97,15 @@ public function queryAction(Request $request)
         $hits = $this->searchManager
             ->createSearch($queryString)
             ->locale($locale)
-            ->index('page_' . $webspace->getKey() . '_published')
+            ->indexes(
+                \str_replace(
```

Comment:
I personally would move this logic into the `SuluSearchExtension.php`

Reply:
@alexander-schranz just the `str_replace`? or also the `array_filter` and `array_values`? because then the keys are lost

Reply:
Yes would move all into the Extension. Think the controller should not know about the key as that is configuration logic.

Reply:
@alexander-schranz I just realized, that the current webspace isn't available when setting the parameter

Reply:
Thats correct would still move the other logic into the Extension.